core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -120,6 +120,12 @@ object OptionT extends OptionTInstances {
     def apply[A](value: Option[A])(implicit F: Applicative[F]): OptionT[F, A] =
       OptionT(F.pure(value))
   }
+
+  /**
+    * Lifts the `F[A]` Functor into an `OptionT[F, A]`.
+    *
+    */
+  def liftF[F[_], A](fa: F[A])(implicit F: Functor[F]): OptionT[F, A] = OptionT(F.map(fa)(Option(_)))
```

Comment:
This may not really be something to worry about, but I'm thinking that maybe this should use `Some(_)` instead of `Option(_)`. The `Option.apply` method has logic to turn `null` into `None`. In Cats we are generally assuming that people aren't passing around `null`. But if for some reason they really want to lift a `null` value into the `OptionT` structure, I don't think we should prevent them from doing that here.